src/sentry_plugins/github/webhooks/events/pull_request.py

```diff
@@ -53,16 +54,13 @@ def __call__(self, event, organization):
             )
             author_email = commit_author.email
         except CommitAuthor.DoesNotExist:
-            try:
-                user_model = User.objects.filter(
-                    social_auth__provider="github",
-                    social_auth__uid=user["id"],
-                    org_memberships=organization,
-                )[0]
-            except IndexError:
-                pass
-            else:
-                author_email = user_model.email
+            for social_auth in UserSocialAuth.objects.filter(provider="github", uid=user["id"]):
+                member = OrganizationMember.objects.filter(
+                    organization_id=organization.id, user_id=social_auth.user.id
+                ).first()
```

Comment:
This will be ideally the org member mapping table in the future. This is a workaround to be able to drop `Organization.members` column.